libraries/src/Form/Field/ModuleorderField.php

```diff
@@ -43,7 +43,7 @@ class ModuleorderField extends FormField
      * The linked property
      *
      * @var    string
-     * @since  __DEPLOY_VERSION_
+     * @since  __DEPLOY_VERSION__
```

Comment:
```suggestion
     * @since  4.2.7
```
Set to fixed version because Joomla 4.2.7 is already released

Reply:
Thank you, you are right!